[bug]: NPM ERR

### What version of strapi are you using?

4.9.1

### What version of strapi-plugin-media-browser are you using?

latest

### What's Wrong?

Dependency error when using NPM to install. 

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: testlibrary@0.1.0
npm ERR! Found: react-is@18.2.0
npm ERR! node_modules/react-is
npm ERR!   react-is@"^18.0.0" from react-redux@8.0.5
npm ERR!   node_modules/react-redux
npm ERR!     react-redux@"8.0.5" from @strapi/admin@4.9.1
npm ERR!     node_modules/@strapi/admin
npm ERR!       @strapi/admin@"4.9.1" from @strapi/strapi@4.9.1
npm ERR!       node_modules/@strapi/strapi
npm ERR!         peer @strapi/strapi@"^4.3.4" from @strapi/admin@4.9.1
npm ERR!         7 more (@strapi/data-transfer, ...)
npm ERR!     react-redux@"8.0.5" from @strapi/plugin-content-type-builder@4.9.1
npm ERR!     node_modules/@strapi/plugin-content-type-builder
npm ERR!       @strapi/plugin-content-type-builder@"4.9.1" from @strapi/strapi@4.9.1
npm ERR!     4 more (@strapi/plugin-documentation, @strapi/plugin-i18n, ...)
npm ERR!   peer react-is@">= 16.8.0" from styled-components@5.3.3
npm ERR!   node_modules/styled-components
npm ERR!     styled-components@"5.3.3" from @strapi/admin@4.9.1
npm ERR!     node_modules/@strapi/admin
npm ERR!       @strapi/admin@"4.9.1" from @strapi/strapi@4.9.1
npm ERR!       node_modules/@strapi/strapi
npm ERR!         peer @strapi/strapi@"^4.3.4" from @strapi/admin@4.9.1
npm ERR!         7 more (@strapi/data-transfer, ...)
npm ERR!     peer styled-components@"^5.2.1" from @strapi/design-system@1.6.6
npm ERR!     node_modules/@strapi/design-system
npm ERR!       @strapi/design-system@"1.6.6" from @strapi/admin@4.9.1
npm ERR!       node_modules/@strapi/admin
npm ERR!         @strapi/admin@"4.9.1" from @strapi/strapi@4.9.1
npm ERR!         node_modules/@strapi/strapi
npm ERR!       8 more (@strapi/helper-plugin, ...)
npm ERR!     9 more (@strapi/helper-plugin, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! strapi-plugin-media-browser@"*" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: react-is@17.0.2
npm ERR! node_modules/react-is
npm ERR!   peer react-is@"^17.0.0" from strapi-plugin-media-browser@0.0.1
npm ERR!   node_modules/strapi-plugin-media-browser
npm ERR!     strapi-plugin-media-browser@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Had to use --legacy-peer-deps.

Then when trying to run build i get this:

`Error: Plugin name "Media Browser" is not in kebab (an-example-of-kebab-case)`

### To Reproduce

Just use npm install

### Expected Behaviour

no react dependency issues 

### Link to repo

n/a

--- a/package.json
+++ b/package.json
@@ -89,9 +89,9 @@
   },
   "peerDependencies": {
     "@strapi/strapi": ">=4.0.0",
-    "react": "^17.0.0",
-    "react-dom": "^17.0.0",
-    "react-is": "^17.0.0",
+    "react": "^17.0.0 || ^18.0.0",
+    "react-dom": "^17.0.0 || ^18.0.0",
+    "react-is": "^17.0.0 || ^18.0.0",
     "react-router-dom": "^5.0.0",
     "styled-components": "^5.0.0"
   },

